### DISCUSSION TITLE
RTL (Right-to-Left) in Winforms?

### DISCUSSION THREAD

**Discussion by kachnitata** [BlueMystical/Dark-Mode-Forms#28] [Ideas]
I am just wondering if anoyne of you guys came across an RTL environment in Winforms.
Today I realized my recent contribution to Dark-Mode-Forms would require a small adjustment to work both in LTR and RTL.
But I don't know whether the rest of the library is RTL-compliant at all and whether I should try.
I also currently have no environment to debug and test it.

**Comment by BlueMystical** [comment]
No, i didnt made it with RTL in mind.
But i had not verified it, so it could be, idk.
